service_ui/src/Components/Results/BaseResults.tsx

```diff
@@ -101,17 +110,25 @@ export const BaseResults: React.FC<baseResultsProps> = ({QueryData, includedKeyw
         )
       }
   
-      else return (
-      <>
-      <Background />
-        <div className="App">
-          <button className='go-back-button' onClick={()=>{setPageNumber(0)}}>go back</button>
-          <div className='flex result'>
-            <h3 className='result-header'> Example Results</h3>
-            <h4 className='result-sub-header'>keyword check in a hardcoded HTML from ynet</h4>
-            <ImageGallery items={images} showThumbnails showFullscreenButton showNav/>
-          </div>
+      else return ( 
+      loading? (
```

Comment:
do you think we should keep background on loading page? or maybe change it a bit? or not have one at all?
also we could delete this check and just have a loading page show up automatically now, instead of having a third option (the section under this)